lang/de/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: german
+title: Semantic Versioning 2.0.0
+---
+
+Semantic Versioning 2.0.0
+======================
+
+Zusammenfassung
+-------------
+
+Auf Grundlage einer Versionsnummer von MAJOR.MINOR.PATCH werden die einzelnen Elemente folgendermaßen erhöht:
+
+1.  MAJOR wird erhöht, wenn API-inkompatible Änderungen veröffentlicht werden,
+2.  MINOR wird erhöht, wenn neue Funktionalitäten, welche kompatibel zur bisherigen API sind, veröffentlicht werden, und
+3.  PATCH wird erhöht, wenn die Änderungen ausschließlich API-kompatible Bugfixes umfassen.
+
+Außerdem sind Bezeichner für Pre-Releases und Build Metadaten als Erweiterungen zum MAJOR.MINOR.PATCH Format verfügbar.
```

Comment:
I think adding line comments are fine for now. Let's see how many you add. A PR into this PR is also fine. Really up to the original submitter.